content/blog/getting-started-to-generate-dscc-api-client-libraries-for-python-using-openapi-generator.md

```diff
@@ -0,0 +1,217 @@
+
+```shell
+~$ java -jar openapi-generator-cli.jar generate -i storage-api.yaml -g python -o sdks/dscc-python-sdk
+```
+
+* 'i' represents the input file, that is the OpenAPI spec which can be in the form of JSON or YAML.
+* 'generate' represents generating the code based on the specified generator
+* 'g' represents the generator/language name like Java, Go.
+* 'o' represents output directory where the client library will be generated.
+
+This Python DSCC client library can be generated in a few minutes. Below is a snippet of screen output during the generation a Python client library using the openapi-generator.jar file.
+
+![](/img/client-generation.jpg "Generating Python SDK using OpenAPI generator")
+
+The generated client library can be made available in your workstation, or it can also be uploaded to a GitHub library so that it can be made available for others to use. An example of the GitHub repository of a sample Python client library looks like this: 
+
+![](/img/python-open-api-sdk-repo.jpg "GitHub repository of Python Client library generated using OpenAPI generator")
+
+The client library comes with the following:
+
+* The code for assisting the DSCC API calls.
+* Documentation for this DSCC client library that derived from the API spec.
+* Tests codes that can be used to validate the operation of this client library.
+* Examples of every endpoint available in the README.md
+* Required python dependencies (requirements.txt and test-requirements.txt) that are required for using this SDK
+
+For instance, take a look at the README file of this repository which has the following info.
+
+1. Installation instructions
+
+![](/img/readme.png "Python SDK installation instructions")
+
+2. A sample code to get started with 
+
+![](/img/sample.png "Sample code")
+
+3. Documentation list for all endpoints
+
+![](/img/api-endpoints.png "List of endpoints")
+
+4. Documentation list for all models
+
+![](/img/models.png "List of models")
+
+5. Documentation about authorization
+
+![](/img/auth.png "Authorization of API calls")
+
+Lets run a sample code which displays the access types in DSCC. The usage of each endpoint is given under documentation of each endpoint in the form of a sample code. To execute the operations on the DSCC, all that is needed for a user to do is:
+
+1. Provide an authorized token which is generated from the HPE GreenLake as mentioned in the [blog](https://developer.hpe.com/blog/oauth2-for-hpe-greenlake-data-services-cloud-console/)
+2. Install the Python dependencies using the following command.
+
+```shell
+~$ pip install requirements.txt 
+```
+
+Below is the sample of the code to get the information about the associated user's RBAC association. This code will provide list of the capabilities (port.read, volume.create) of that related user who provided the authorization can exercise. To execute this code, please substitute the YOUR_BEARER_TOKEN with the access token generated in your example.
```

Comment:
I think you may be missing a word or two in the first sentence:
"...the sample of the code **that is used** to get the information"
Add "a" prior to "list of the capabilities"
Actually, better to rephrase that second sentence to:
"This code will provide the capabilities (port.read, volume.create) afforded to the authorized user."